### ISSUE TITLE
GroovyScript error when updating to Labs 8.11

### ISSUE THREAD

**Issue by GavinFF-SS** [Nomi-CEu/Nomi-CEu#906]
### Nomi CEu Version

1.7-beta-6

### Launcher

Prism Launcher

### Configurations or Mods Changed

onDemandAnimatedTextures set to false in the LoliASM config.
Added AE2FCR 2.6.4-r
Added Fugue and Scalar

### Environment

Singleplayer
CleanroomMC

### Mode

Normal Mode

### What Happened

(also, my bad if this issue belongs in the Nomi Labs issues and not here) When updating to Nomi Labs 8.11, GroovyScript errors out, and some AE2 related recipes no longer have a custom recipe.
![2024-08-13_17 14 39](https://github.com/user-attachments/assets/b729d121-ac61-41ae-996d-1d69acd88431)

### Expected Behavior

For Groovy to not be bugged and actually work.

### Reproduction Steps

1. Update Labs to 8.11.
2. Load a world.
3. Some recipes are broken.

### Logs

[groovy.log](https://github.com/user-attachments/files/16605429/groovy.log)

### Additional Information

I did try deleting the nomilabs configuration, since I saw something that could've been causing the error, but it did nothing.

**Comment by IntegerLimit** [comment]
This is known, its just a simple change in the GrS script that needs to be made when changing to labs 0.8.11.

**Comment by GavinFF-SS** [comment]
what needs to be modified to fix this error?

**Comment by FJSAGS** [comment]
also recipes dont change when updating to 8.11
![image](https://github.com/user-attachments/assets/b00df3e4-bdfa-4909-b64c-48f3cb1ed8d1)

**Comment by MatthiasMann** [comment]
In the file `groovy/postInit/Post-Initial/Main/General/Misc/tooltips.groovy` line 253 change to:
```groovy
	int xpPercent = (matter.xpValue / LabsConfig.advanced.otherModsLinearXp) * 100
```

**Comment by FJSAGS** [comment]
![image](https://github.com/user-attachments/assets/e60302ef-be1d-42fd-953f-dfa99b0e3ef4)
it found another error

**Comment by FJSAGS** [comment]
![image](https://github.com/user-attachments/assets/d9918a14-ee0c-4f19-a4de-7f2e9dd4aa7f)
The same line i think

**Comment by MatthiasMann** [comment]
Yes, that's why you need to edit the config file

**Comment by FJSAGS** [comment]
Now it works! Thank you

**Comment by IntegerLimit** [comment]
Manual update no longer needed, simply download the latest nightly release once the build is finished (in around 2-3 minutes), and beta-6a when that releases today.
